packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -75,6 +75,7 @@ abstract class IrisGridModel<
     RECONNECT: 'RECONNECT',
     TOTALS_UPDATED: 'TOTALS_UPDATED',
     PENDING_DATA_UPDATED: 'PENDING_DATA_UPDATED',
+    VIEWPORT_UPDATED: 'VIEWPORT_UPDATED',
```

Comment:
Let's add some JSDocs for this event. In retrospect, we should have them for all of the events, but we can at least be proactive and add some here to make it clear what the event is for. 

When writing good docs for a function, you want to address possible questions a future reader for have; think about what kind of questions a reader of this code (or user of the event) may have. 

For example, is this fired when the set viewport in the model is changed, when the viewport is applied to the table, or when the viewport data has come back? Is there any data included with the event? Another common question is "Why?", as in "Why do we need this event?"
```suggestion

    /** Fired when the viewport is applied to the table and we're waiting for a response. */
    VIEWPORT_UPDATED: 'VIEWPORT_UPDATED',
```